packages/client/store/store.js

```diff
@@ -2,7 +2,9 @@ import { create } from "zustand";
 
 const useStore = create(set => ({
     geoData: {lat: -8.7445, lng: 115.182},
-    // setGeoData: (location) => set(state => state.geoData = location)
+    markers: [],
+    setMarkers: (object) => set((state) => ({markers: [...state.markers, object]})),
+    setGeoData: (location) => set(state => state.geoData = location)
```

Comment:
Not sure if this should work as you're mutating the state and that property `geoData` is a object. Be careful of mutating state and objects/arrays. Look at how `shallow` checks are preformed differently for primitives vs objects/arrays in state update check like zustand or just react. 

```suggestion
    setGeoData: (location) => set({geoData: location})
```